src/shared/exec-util.h

```diff
@@ -48,9 +48,9 @@ typedef enum ExecCommandFlags {
         EXEC_COMMAND_IGNORE_FAILURE   = 1 << 0,
         EXEC_COMMAND_FULLY_PRIVILEGED = 1 << 1,
         EXEC_COMMAND_NO_SETUID        = 1 << 2,
-        EXEC_COMMAND_AMBIENT_MAGIC    = 1 << 3,
-        EXEC_COMMAND_NO_ENV_EXPAND    = 1 << 4,
+        EXEC_COMMAND_NO_ENV_EXPAND    = 1 << 3,
         _EXEC_COMMAND_FLAGS_INVALID   = -EINVAL,
+        _EXEC_COMMAND_FLAGS_MAX       = (1 << 4) -1,
```

Comment:
Hmm, is this really useful? And at least this should be `_ALL`, not `_MAX`.

Reply:
yes, it is useful. I added an assert_cc() to exec-util.c that makes use of this and ensures the associated string array is complete.